README.md

```diff
@@ -323,25 +323,13 @@ A large portion of the Windows support comes from code by [Krzysztof Kowalczyk](
 Prebuilt binaries
 -----------------
 
-LevelDOWN uses `prebuild` to support prebuilt binaries. See [this list](https://github.com/Level/leveldown/releases) of supported prebuilt platform binaries. When installing LevelDOWN `prebuild --download` will download prebuilt binaries from GitHub if they exist and fallback to a compile step if they don't.
-
-If you are working on LevelDOWN and want to compile the C++ code you can do:
-
-```
-$ npm run rebuild [--debug] [--verbose]
-```
-
-or
-
-```
-$ npm i --build-from-source [--debug] [--verbose]
-```
+LevelDOWN uses `prebuild` and `prebuild-install` for handling prebuilt binaries. See [this list](https://github.com/Level/leveldown/releases) of supported prebuilt platform binaries. When installing LevelDOWN `prebuild-install` will install prebuilt binaries from GitHub if they exist and fallback to a compile step if they don't.
```

Comment:
@juliangruber it's enough with `npm install`, not sure if that's needed to document for people hacking on leveldown ..